source/boundary_composition/box.cc

```diff
@@ -174,6 +174,7 @@ namespace aspect
     ASPECT_REGISTER_BOUNDARY_COMPOSITION_MODEL(Box,
                                                "box",
                                                "A model in which the composition is chosen constant on "
-                                               "all the sides of a box.")
+                                               "the selected sides of a box. Sides otherwise default to "
+                                               "zero flux conditions.")
```

Comment:
I think this should specifically specify what it means for a face to be "selected". You can list the specific parameter in the input file to make this clear.